### ISSUE TITLE
Seperability of points 

### ISSUE THREAD

**Issue by anujgupta82** [distillpub/post--misread-tsne#3]
This is not a bug but query. 

While the writeup address some important question, it missed one question. 

In ML/DS people want to know if their higher dimensional Data is separable or not. It will be great if there can be some insights on the use t-SNE for this

**Comment by zihaolucky** [comment]
Agree, it's useful to visualize feature representation using t-SNE.

@anujgupta82 I think maybe the last one on the second row and the first one in third row are want you want.

**Comment by colah** [comment]
Since t-SNE is largely trying to preserve the local structure of the data, one might not expect it to be every helpful for seeing linear separability.

This example shows that t-SNE can take non-linearly separable data and present it as linearly separable:

![image](https://cloud.githubusercontent.com/assets/61658/24261312/d4a87444-0fb3-11e7-9c9c-92a2587e645f.png)

That said, will give opportunities for authors to comment. CC @wattenberg
